homeassistant/components/fritz/device_tracker.py

```diff
@@ -143,7 +143,7 @@ def source_type(self) -> str:
         return SOURCE_TYPE_ROUTER
 
     @property
-    def device_info(self) -> Dict[str, any]:
+    def device_info(self) -> dict[str, any]:
```

Comment:
```suggestion
    def device_info(self) -> dict[str, Any]:
```
You would also need to add `from typing import Any`